docs/data/material/pages.ts

```diff
@@ -236,6 +236,11 @@ const pages = [
       { pathname: '/material-ui/discover-more/languages' },
     ],
   },
+  {
+    pathname: 'https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=sidenav',
+    title: 'Templates',
```

Comment:
```suggestion
    title: 'Store',
```

Should we use _Store_ given that the Design kits are there as well and, therefore, it's more than the Templates?

Reply:
Wouldn't "Templates" and "Design kits" convert better than "Store"?
IMHO "Store" is too abstract and produces doubts like "Is it a merch store?" and "Is there something I might need?", while "Templates" and "Design kits" are straightforward

Reply:
We already link the design kits from each page:

<img width="778" alt="Screenshot 2022-05-06 at 14 00 21" src="https://user-images.githubusercontent.com/3165635/167127427-102a8a60-f00f-48f5-8d96-a49054736870.png">

I think that it's the templates that we care about here.

Reply:
The Design kits page is inside the DIscover more folder — should we move it outside from it?

Reply:
No preferences, I will fix this https://github.com/mui/material-ui/pull/32180#issuecomment-1120264250 now though. I'm merging and deploying now as each day costs us > $1,000.